test/cypress/support/commands.js

```diff
@@ -336,6 +367,62 @@ Cypress.Commands.add('giAcceptGroupInvite', (invitationLink, {
   }
 })
 
+Cypress.Commands.add('giAcceptUsersGroupInvite', (invitationLink, {
```

Comment:
Why are there two different commands that essentially do the same exact thing?

Can you see if you can DRY `giAcceptGroupInvite` and `giAcceptUsersGroupInvite` by combining them together?

Reply:
Okay. I will combine them together.

Reply:
@taoeffect, I tried to combine them together, and started to think that we should keep both of them with the reasons below;
- We should change argument names (`username` -> `usernames`, `displayName` -> `displayNames`), and it drives many changes in all the test scenario files
- We have three arguments (`isLoggedIn`, `shouldLogoutAfter`, `inviteCreator`) which are only for `giAcceptGroupInvite` function which should be used for single-user joining, not for multi-user joining. If we combine two functions together, these three arguments should be checked when it is multi-user joining, and that could be messy I guess.
- `existingMemberUsername` could be optional in single-user joining when he joins the group by the second time or more.

Reply:
@Silver-IT OK if you think we should keep them separate then we can. In that case can you please add a comment above each command explaining what it does and how it's different? Also, maybe rename one of the commands to make it more clear how it's different from the other.